src/shared/ServiceCatalog.ts

```diff
@@ -1,30 +1,11 @@
 import axios from "axios";
 
 import * as urls from "../shared/url";
-import { AppRepository } from "./AppRepository";
 import { IClusterServiceClass } from "./ClusterServiceClass";
 import { IServiceInstance } from "./ServiceInstance";
-import { IAppRepository, IK8sList, IStatus } from "./types";
+import { IK8sList, IStatus } from "./types";
 
 export class ServiceCatalog {
-  public static async getCatalogRepo(): Promise<IAppRepository | undefined> {
```

Comment:
these methods are not being used, the svc-cat chart repo is being installed by default